### PR TITLE
[now-cli] Use `devCommand` from project settings locally

### PR DESCRIPTION
**Pull request by lucleray**
Example:
image

The `devCommand` from project settings is used if:
- the project is linked (`.now` exists)
- `project.devCommand` is not `null`
- `project.devCommand` is null but `project.framework` is not `null`

When `devCommand` is used, these builders are filtered out:
- `@now/static-build`
- `@now/next`

### CODE REVIEW

File: packages/now-cli/src/util/dev/server.ts [hunk 1]
```diff
@@ -1547,6 +1605,55 @@ export default class DevServer {
...
+  async runDevCommand() {
+    if (!this.devCommand) return;
+
+    const cwd = this.cwd;
+
+    const { stdout: yarnBinStdout } = await execAsync('yarn', ['bin'], {
+      cwd,
+    });
+
+    const nodeBinPath = yarnBinStdout.trim();
```
Comment (TooTallNate) [inline packages/now-cli/src/util/dev/server.ts]:
```suggestion
    const yarnBinPath = yarnBinStdout.trim();
```

File: packages/now-cli/src/util/dev/server.ts [hunk 2]
```diff
@@ -1547,6 +1605,55 @@ export default class DevServer {
...
+    this.output.log(
+      `Running Dev Command ${chalk.cyan.bold(`“${this.devCommand}”`)}`
+    );
+
+    const port = await getPort();
+
+    const env: EnvConfig = {
+      ...process.env,
+      ...this.buildEnv,
+      PATH: `${nodeBinPath}${delimiter}${process.env.PATH}`,
```
Comment (TooTallNate) [inline packages/now-cli/src/util/dev/server.ts]:
```suggestion
      PATH: `${yarnBinPath}${delimiter}${process.env.PATH}`,
```

File: packages/now-cli/src/util/dev/server.ts (hunk 1 above)
Comment (lucleray) [reply packages/now-cli/src/util/dev/server.ts]: It's still the `node_modules` bin folder 🤔 @TooTallNate

Comment (lucleray) [reply packages/now-cli/src/util/dev/server.ts]: I don't really mind though. It looks good too, I'll update 👍

File: packages/now-cli/src/commands/dev/dev.ts [hunk 3]
```diff
@@ -20,7 +24,40 @@ export default async function dev(
   const cwd = path.resolve(dir);
   const listen = parseListen(opts['--listen'] || '3000');
   const debug = opts['--debug'] || false;
-  const devServer = new DevServer(cwd, { output, debug });
+
+  // let devCommand = 'hexo server --port $PORT';
```
Comment (TooTallNate) [inline packages/now-cli/src/commands/dev/dev.ts]: Remove?

File: packages/now-cli/src/util/dev/server.ts [hunk 4]
```diff
@@ -861,6 +882,19 @@ export default class DevServer {
       ops.push(shutdownBuilder(match, this.output));
     }
 
+    ops.push(
+      new Promise(resolve => {
+        if (!this.devProcess) {
+          resolve();
```
Comment (TooTallNate) [inline packages/now-cli/src/util/dev/server.ts:888]:
```suggestion
          resolve();
          return;
```

File: packages/now-cli/src/util/dev/server.ts [hunk 5]
```diff
@@ -861,6 +882,19 @@ export default class DevServer {
...
+        }
+
+        if (this.devProcess) {
```
Comment (TooTallNate) [inline packages/now-cli/src/util/dev/server.ts]: If you apply the suggestion above then this `if` branch can be removed.

```suggestion
```

**Comment by peadrakw** [comment]
⬇️ Suggested change
-        if (this.devProcess) {

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**